Tests/SwiftLintFrameworkTests/FunctionBodyWhitespaceCommentRuleTests.swift

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
Consider using triggering and nontTriggering examples when defining the rule (see other rules for example). With triggering and nontTriggering SwiftLint will:

* Add examples to the Rules.md file
* Build and run testes automatically looking for violations (making this test file unnecessary).

Reply:
@ornithocoder I have changed the tests file. Now it is similar to FuctionBodyLengthRuleTests.